src/server/pkg/storage/chunk/chunk.proto

```diff
@@ -6,7 +6,7 @@ option go_package = "github.com/pachyderm/pachyderm/src/server/pkg/storage/chunk
 // DataRef is a reference to data within a chunk.
 message DataRef {
   // The chunk the referenced data is located in.
-  ChunkInfo chunk_info = 1;
+  ChunkRef chunk_ref = 1;
```

Comment:
I know I said this change is fine, but I am not sure about it now. It seems like it could be confusing since we call data refs chunk refs when they refer to the full chunk. Why did you want to change this? Something to note is that the ChunkInfo / Chunk pattern lines up pretty well with the general proto naming convention we follow in pfs / pps (i.e. CommitInfo / Commit).

Reply:
The identifying information for the chunk is not going to change from it's hash.  So we are never going to realize the benefit of having a Chunk object that we can pack more information into over time. I call it a Ref because I envision it being exchanged for chunk data by Create and Get functions which will deal with compression/encryption, and operate on a chunk.Client.